packages/storybook/src/spright/chat/message/chat-message.stories.ts

```diff
@@ -45,6 +68,28 @@ export const chatMessageText: StoryObj<ChatMessageTextArgs> = {
     render: createUserSelectedThemeStory(html`
```

Comment:
This indentation seems surprising in this section. You can start this section with prettier-ignore to manually control the indentation

```suggestion
    // prettier-ignore
    render: createUserSelectedThemeStory(html`
```